fluentbit/conf/conf.go

```diff
@@ -255,8 +255,7 @@ const (
     # https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver
     Name              stackdriver
     resource          gce_instance
-    stackdriver_agent {{.UserAgent}}
-    workers           {{.Workers}}
+    stackdriver_agent {{.UserAgent}}{{.Workers}}
```

Comment:
Could you just do this instead?

    {{if .Workers}}
    workers {{.Workers}}
    {{end}}

Reply:
Yeah, that would be much better than the code change in confgenerator.go.

You can even get slightly shorter:
```
{{with .Workers}}
workers {{.}}
{{end}}
```

Reply:
Changed to 

```
    stackdriver_agent {{.UserAgent}}{{if .Workers}}
    workers           {{.Workers}}{{end}}
```

Instead. It looks a bit weird because I had to make it on the same line to avoid it generating an additional new line.